packages/react-charts/src/victory/components/ChartArea/examples/ChartAreaBasic.tsx

```diff
@@ -0,0 +1,68 @@
+import { Chart, ChartArea, ChartAxis, ChartGroup, ChartVoronoiContainer } from '@patternfly/react-charts/victory';
+// import '@patternfly/patternfly/patternfly-charts.css'; // Required for mix-blend-mode CSS property
+
+interface PetData {
+  name: string;
+  x: string;
+  y: number;
+}
+
+export const ChartAreaRightAlignedLegend: React.FunctionComponent = () => {
```

Comment:
I think ChartAreaRightAlignedLegend would make for a good file name as well rather than ChartAreaBasic.

Also can we look into removing the commented out imports in these files